Source/LibationWinForms/Form1.Liberate.cs

```diff
@@ -7,11 +9,14 @@ public partial class Form1
 	{
 		private void Configure_Liberate() { }
 
-		private async void beginBookBackupsToolStripMenuItem_Click(object sender, EventArgs e)
-			=> await BookLiberation.ProcessorAutomationController.BackupAllBooksAsync();
+		//GetLibrary_Flat_NoTracking() may take a long time on a hugh library. so run in new thread 
+		private void beginBookBackupsToolStripMenuItem_Click(object sender, EventArgs e)
+			=> Task.Run(() => processBookQueue1.AddDownloadDecrypt(ApplicationServices.DbContexts.GetLibrary_Flat_NoTracking()
```

Comment:
[Eliding](https://blog.stephencleary.com/2016/12/eliding-async-await.html) should be used carefully and I'm wary of using it with UI event delegates. If you've tested these this runs correctly as async, we can leave it though.